docs/configuration/OVERVIEW.md

```diff
@@ -36,22 +36,23 @@ A component will have to handle different types of configurations.
 
 ### Default configuration
 
-- Each __component type__ will have a default configuration which will be defined in _.config.ts_ file associated to the
+- Each __configurable component__ will have a default configuration which will be defined in _.config.ts_ file associated to the
   component
 
-> __WARNING__ the field name 'id' should not be used in the configuration, as we created a unique one for the entity configuration store
+> [!WARNING]
+> The field name 'id' should not be used in the configuration, as we created a unique one for the entity configuration store.
 
 ### Configuration coming from config store
 
-- Each __component type__ can have a customized config (compliant with the store model). This custom config can be
+- Each __configurable component__ can have a customized config (compliant with the store model). This custom config can be
   asynchronously loaded from a backend server, or it can be injected in the body tag of the app. In both cases, this
   content will be used to update the store.
 - If present, it will override the default config. (Don't worry about the implementation now, we'll see it later)
 
 ### Global config
 
 - The common configuration is the one used in multiple components (it can be a date format, price display, type of input
-  form fields from an angular material input element ...)
+  form fields from an Angular material input element ...)
 - A common configuration is defined in every library. The application common configuration (__global config__) will be
```

Comment:
__merge of all common__ configurations. => __merge of all the common configurations__. ?